README.md

```diff
@@ -167,6 +167,8 @@ The current stable release is [1.2.4](https://github.com/ruby-grape/grape/blob/v
 
 ## Installation
 
+After **1.2.5**, Ruby 2.4.0 or newer is required.
```

Comment:
This should go into UPGRADING, for this doc just state that Ruby 2.4 or newer is required.